language/rust/develop.md

```diff
@@ -96,13 +96,88 @@ For the sample application, you'll use a variation of the backend from the react
    ? What port does your server listen on? 8000
    ```
 
-3. In the cloned repository's directory, run `docker build` to build the image.
+3. In the cloned repository's directory, open the `Dockerfile` in an IDE or text editor to update it.
+
+   `docker init` handled creating most of the instructions in the Dockerfile, but you'll need to update it for your unique application. In addition to a `src` directory, this application includes a `migrations` directory to initialize the database. Add a bind mount for the `migrations` directory to the build stage in the Dockerfile. The following is the updated Dockerfile.
```

Comment:
**Thought:** Is there a way our styling for code snippets can include line numbers? That way it's easier to identify what line needs to be changed. It was easy enough to compare the original with the updated one here, but just thought that would make it extra clear.

Reply:
No way to include line numbers currently.
One alternative we have used is specifying the line number and using a diff codeblock. It doesn't work good with copy-paste though.
For example, https://docs.docker.com/get-started/03_updating_app/

Reply:
Gotcha, this works as-is then I would say!